pkgs/applications/science/logic/klee/default.nix

```diff
@@ -37,12 +35,8 @@
 let
   # Python used for KLEE tests.
   kleePython = python3.withPackages (ps: with ps; [ tabulate ]);
-
-  # The klee-uclibc derivation.
-  kleeuClibc = callPackage ./klee-uclibc.nix {
-    inherit stdenv clang llvm extraKleeuClibcConfig debugRuntime runtimeAsserts;
-  };
-in stdenv.mkDerivation rec {
+in
+llvmPackages.stdenv.mkDerivation (final: rec {
```

Comment:
Why `final: rec {` instead of just `rec`?

Reply:
It was to facilitate the passthru but I can remove this.